<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IndexWidget</class>
 <widget class="QWidget" name="IndexWidget">
  <property name="minimumSize">
   <size>
    <width>240</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>IndexWidget</string>
  </property>
  <layout class="QVBoxLayout" name="indexWidgetLayout">
   <property name="spacing">
    <number>20</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item>
    <spacer name="vspacer1">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>195</width>
       <height>7</height>
      </size>
     </property>
    </spacer>
   </item>
   <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
    <widget class="QLabel" name="logoLabel">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../Icons/RRTFLogo.png</pixmap>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="vspacer2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>195</width>
       <height>17</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="schedulesButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>44</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	color: rgb(0, 0, 0);
	border: none;
}

QPushButton::checked {
	background-color:rgb(75, 110, 175);
}</string>
     </property>
     <property name="text">
      <string> Schedules</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>D:/jacbo/Downloads/calendar.png</normaloff>D:/jacbo/Downloads/calendar.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="ordersButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>44</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	color: rgb(0, 0, 0);
	border: none;
}

QPushButton::checked {
	background-color:rgb(75, 110, 175);
}</string>
     </property>
     <property name="text">
      <string> Orders</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>D:/jacbo/Downloads/clipboard.png</normaloff>D:/jacbo/Downloads/clipboard.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>23</width>
       <height>23</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="driversButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>44</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	color: rgb(0, 0, 0);
	border: none;
}

QPushButton::checked {
	background-color:rgb(75, 110, 175);
}</string>
     </property>
     <property name="text">
      <string> Drivers</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>D:/jacbo/Downloads/driver.png</normaloff>D:/jacbo/Downloads/driver.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>21</width>
       <height>21</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="vehiclesButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>44</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	color: rgb(0, 0, 0);
	border: none;
}

QPushButton::checked {
	background-color:rgb(75, 110, 175);
}</string>
     </property>
     <property name="text">
      <string> Vehicles</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>D:/jacbo/Downloads/vehicle.png</normaloff>D:/jacbo/Downloads/vehicle.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="settingsButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>44</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	color: rgb(0, 0, 0);
	border: none;
}

QPushButton::checked {
	background-color:rgb(75, 110, 175);
}</string>
     </property>
     <property name="text">
      <string> Settings</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>D:/jacbo/Downloads/settings.png</normaloff>D:/jacbo/Downloads/settings.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="vspacer3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>195</width>
       <height>47</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="aboutButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>44</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	color: rgb(0, 0, 0);
	border: none;
}

QPushButton::checked {
	background-color:rgb(75, 110, 175);
}</string>
     </property>
     <property name="text">
      <string>About</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>D:/jacbo/Downloads/questionMark.png</normaloff>D:/jacbo/Downloads/questionMark.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>18</width>
       <height>18</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="vspacer4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>195</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <spacer name="vspacer5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
